common/client/multi_node_test.go

```diff
@@ -666,49 +667,57 @@ func TestMultiNode_SendTransaction(t *testing.T) {
 			sendTxSoftTimeout:   tests.TestInterval,
 		})
 		err := mn.SendTransaction(tests.Context(t), nil)
-		require.NoError(t, err)
+		require.EqualError(t, err, expectedError.Error())
 	})
-	t.Run("Fails when closed on sendonly broadcast", func(t *testing.T) {
+	t.Run("Returns success without waiting for the rest of the nodes", func(t *testing.T) {
```

Comment:
Could you ensure that we call mn.Close() at the end of all tests?
This will help test that the MultiNode's waitgroups have all ended gracefully, especially with more dependencies added now on the wg field.

Reply:
All of the subtests explicitly close multinode at the end of case or we do it on cleanup:

```golang
newStartedMultiNode := func(t *testing.T, opts multiNodeOpts) testMultiNode {
		mn := newTestMultiNode(t, opts)
		err := mn.StartOnce("startedTestMultiNode", func() error { return nil })
		require.NoError(t, err)
		t.Cleanup(func() {
			require.NoError(t, mn.Close())
		})
		return mn
	}
```